pkg/backup.go

```diff
@@ -184,42 +187,54 @@ func (opt *etcdOptions) backupEtcd(targetRef api_v1beta1.TargetRef) (*restic.Bac
 		return nil, err
 	}
 
-	// init restic wrapper
-	resticWrapper, err := restic.NewResticWrapper(opt.setupOptions)
-	if err != nil {
+	// clear directory before running etcdctl snapshot save
+	klog.Infoln("Cleaning up directory: ", filepath.Join(apis.TmpDirMountPath, "data"))
+	if err := clearDir(filepath.Join(apis.TmpDirMountPath, "data")); err != nil {
```

Comment:
Use a variable named `interimDataDir` in options. Then,  initialize it during creating opt variable.